src/openrct2/ride/coaster/HybridCoaster.cpp

```diff
@@ -1294,7 +1294,7 @@ namespace HybridRC
                     {
                         case 3:
                             PaintAddImageAsParentRotated(
-                                session, direction, GetTrackColour(session).WithIndex(SPR_G2_HYBRID_TRACK_FLAT_DIAGONAL + 3),
+                                session, direction, GetTrackColour(session).WithIndex(SPR_G2_HYBRID_TRACK_FLAT_DIAGONAL + 1),
```

Comment:
? Why this change. Has the order of sprites changed or something?

Reply:
There were redundant sprites. Flat pieces have rotational symmetry so you only need two angles, not four (and vanilla tracks only have two sprites for these pieces), but for some reason I forgot this and had rendered four sprites for the diagonal flat pieces.

I didn't want to make the brake sprites similarly redundant, and I didn't want it to be inconsistent, so I removed the duplicates.